src/components/SortDropdown/SortDropdown.tsx

```diff
@@ -0,0 +1,71 @@
+import { ChevronDownIcon } from '@chakra-ui/icons'
+import type { ButtonProps, MenuProps } from '@chakra-ui/react'
+import {
+  Button,
+  Flex,
+  Menu,
+  MenuButton,
+  MenuItemOption,
+  MenuList,
+  MenuOptionGroup,
+  Text,
+} from '@chakra-ui/react'
+import { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+
+import type { SortOption, SortOptionsKeys } from './types'
+
+type SortDropdownProps = {
+  value: SortOptionsKeys
+  onClick: (arg: SortOptionsKeys) => void
+  buttonProps?: ButtonProps
+  options: SortOption[]
+} & Omit<MenuProps, 'children'>
```

Comment:
Similar to before, we are moving to the below convention to improve performance by reducing re-renders.

```suggestion
type SortDropdownProps = {
  value: SortOptionsKeys
  onClick: (arg: SortOptionsKeys) => void
  buttonProps?: ButtonProps
  options: SortOption[]
  menuProps?: Omit<MenuProps, 'children'>
}
```

Reply:
It was not used, removed it until it's needed [1ac600a](https://github.com/shapeshift/web/pull/8006/commits/1ac600aeb95da5729ef78865c837fd0bcec0f777)